Article/Article.js

```diff
@@ -1,20 +1,23 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// this.expandButton = document.querySelectorAll('.expandButton');
+// expandButton.forEach(expandButton => expandButton.textContent = "expand");
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
-  }
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```

Comment:
try using arrow functions on your actions to not have to worry about binding the this keyword. This can be a pain that you don't have to go through in ES6.